src/modules/Navbar/Navbar.css

```diff
@@ -0,0 +1,87 @@
+.NavbarItems {
+  background: linear-gradient(90deg, rgb(0, 0, 0) 0%, rgb(6, 6, 7) 100%);
+  height: 100px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  font-size: 4rem;
+}
+
+.navbar-logo{
+  position:absolute;
+  left: 50%;
+  margin-left: -50px;  /* 50% of your logo width */
+  display: block;
+  justify-self: center;
+  cursor: pointer;
+}
+
+.search-container{
+  position:relative;
+}
+
+.NavbarItems .search-container input{
+  display:flex;
+  justify-content: left;
+  align-items: center;
+  padding: 20px 10px;
+  background: rgb(126, 124, 124);
+  font-size: 17px;
+  border: solid grey;
+  cursor: pointer;
+}
+
+.NavbarItems .search-container button{
+  position:absolute;
+  right: 20px;
+  top: 10px;
+  border:none;
+  background:none;
+  height:30px;
+  width:30px;
+  border-radius:100%;
+  outline:none;
+  text-align:right;
+  font-weight:bold;
+  padding:2px;
+}
+.fa-react {
+  margin-left: 0.5rem;
+  font-size: 1.6rem;
+}
+
+.nav-menu{
+  display: grid;
+  grid-template-columns: repeat(5,auto);
+  grid-gap: 10px;
+  list-style: none;
+  text-align: center;
+  width: 70vw;
+  justify-content: start;
+  margin-right: 2rem;
+}
+
+.nav-links{
+  color:#EEEEEE;
+  text-decoration: none;
+  padding: 0.5rem 1rem;
+}
+
+.nav-links:hover {
+  background-color:violet;
```

Comment:
Violet doesn't fit with the theme, but we don't really care about the theme right now so that doesn't need to be changed atm.